### DISCUSSION TITLE
Freezing pointer

### DISCUSSION THREAD

**Discussion by YouKnowuwu** [ppy/osu#19902] [Q&A]
When I connect my graphic tablet (wacom intuos CTL-4100) the pointer gets frozen but it's for a few seconds then it moves and it freezes again, and when I connect the mouse it's normal, it moves fluidly.

osu! version: I use osu! lazer, the version is 2022.731.1-lazer

pls help me :c

**Comment by peppy** [comment]
Are you saying your *mouse* becomes weird when the tablet is connected, or your *tablet* is weird but mouse is fine?

**Reply by YouKnowuwu** [reply]
My tablet is weird and the mouse is fine

**Reply by peppy** [reply]
Can you try turning off the tablet support in the game? Search for "tablet" in settings and toggle the enabled checkbox.

**Reply by YouKnowuwu** [reply]
I play with that disabled and it started freezing a long time later. When I activate it, it has no faults but the sensitivity does not work, not even with the one established in the wacom properties.
